packages/plugins/typescript/graphql-apollo/package-lock.json

```diff
@@ -0,0 +1,1942 @@
+{
+  "name": "@graphql-codegen/typescript-graphql-apollo",
```

Comment:
I think you ran npm install on this directory. We use yarn workspaces so we call `yarn test` and `yarn build` on the root.